src/connectors/__init__.py

```diff
@@ -12,6 +12,7 @@
 from . import tenable_settings
 from . import crowdstrike_devices
 from . import cisco_umbrella
+from . import ldap
```

Comment:
I remember regretting the `okta` without `_log` and we have it elsewhere, so might it make more sense to call this `ldap_logs`? no super strong intuition here, but t might be separate from `ldap_settings` or `ldap_accounts`?

Reply:
It was azure.py that we regretted (okta.py still exists), but your point is well taken.